### PR TITLE
RHOARDOC-1478

### PR DESCRIPTION
**Pull request by inoxx03**
- [ ] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [ ] [v2] Have you created a [JIRA](https://issues.jboss.org/browse/THORN) and used it in the commit message?
- [ ] [v4] Have you created a [GitHub Issue](https://github.com/thorntail/thorntail/issues) and used it in the commit message?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/thorntail/thorntail/pulls) for the same issue?
- [ ] Have you built the project locally prior to submission with `mvn clean install`?

-----

### CODE REVIEW

**Comment by manovotn** [comment]
@tradej please take a look at this when you have a minute

ok to test

File: docs/pom.xml [hunk 1]
```diff
@@ -82,6 +82,7 @@
             <product>${swarm.product.build}</product>
             <WildFly>WildFly</WildFly>
             <Thorntail>Thorntail</Thorntail>
+            <open-jdk-builder-image-name>registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift:latest</open-jdk-builder-image-name>
```
Comment (Ladicek) [inline docs/pom.xml]: I don't think the variable names match :-)

Comment (tradej) [reply docs/pom.xml]: I agree, this was probably an oversight from @inoxx03.

**Review by tradej — CHANGES_REQUESTED** [review]

**Review by Ladicek — APPROVED** [review]

**Review by tradej — APPROVED** [review]

**Comment by manovotn** [comment]
Looks good, though it would be nice to have the CI passing.
@Ladicek any clue how to re-trigger Semaphore and which CI job to nudge for that linux build? Apparently it did not listen to my "ok to test" :)

**Comment by Ladicek** [comment]
SemaphoreCI is only for the `4.x` branch, you can ignore it when working with 2.x.

**Comment by Ladicek** [comment]
Looking at the Jenkins server, I have no idea why the Linux build wasn't triggered.

test this please

**Comment by manovotn** [comment]
Tried to temper with some Jenkins settings, let's see if we still get ignored now...

**Comment by manovotn** [comment]
ok to test

**Pull request merged**